common/lib/dependabot/git_commit_checker.rb

```diff
@@ -86,6 +86,10 @@ def head_commit_for_current_branch
       raise Dependabot::GitDependencyReferenceNotFound, dependency.name
     end
 
+    def local_tag_for_branch(name)
+      local_repo_git_metadata_fetcher.head_commit_for_ref(name)
```

Comment:
🤔 I understand `head_commit_for_ref` to return a SHA, which conflicts with my understanding of what `local_tag_for_branch` would return.

I'd expect this method to return a value that has some human-based meaning. It seems like the implementation is what we want, so maybe we could update the method name.

What do you think of something like `#head_commit_for_local_branch` or `#head_commit_for_local_ref`?

Reply:
Indeed! I'm not sure what I had in mind when I used this naming. It doesn't make any sense to me now. Changed to `#head_commit_for_local_branch`, thank you!